tests/test_ase_traj.py

```diff
@@ -43,5 +43,16 @@ def setUp(self):
         self.v_places = 4
 
 
+@unittest.skipIf(skip_ase, "skip ase related test. install ase to fix")
+class TestASEtraj3(unittest.TestCase, CompLabeledSys, IsPBC):
+    def setUp(self):
+        self.system_1 = dpdata.System("ase_traj/Cu.traj", fmt="ase/traj")
+        self.system_2 = dpdata.LabeledSystem("ase_traj/Cu.traj", fmt="ase/traj")
```

Comment:
I've seen the error message in the log. When using `comp_sys.CompLabeledSys`, it's expected to provide two `LabeledSystem`. `comp_sys.CompSys` compares two `System`.

Reply:
so, what i have to do?

Reply:
so what should I do?